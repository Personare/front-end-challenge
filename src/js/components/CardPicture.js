import React from 'react';

const CardPicture = (props) => (
    <div className='CardPicture'>
        <img className='CardPicture__image' src={props.imgUrl} />
    </div>
);

export default CardPicture;
