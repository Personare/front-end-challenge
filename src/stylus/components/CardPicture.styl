.CardPicture
    display inline-block

    +below(breakpoints.phone)
        max-width 100%
        margin 0px

.CardPicture__image
    max-width 162px
    width 100%
    display inline-block

    +below(breakpoints.phone)
        max-width 82px
