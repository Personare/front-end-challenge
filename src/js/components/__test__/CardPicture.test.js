import React from 'react';
import { shallow } from "enzyme";
import CardPicture from '../CardPicture';

describe('CardPicture', () => {

    test('should render correctly', () => {
        const wrapper = shallow(<CardPicture imgUrl='http://via.placeholder.com/162Ã—341' />);
        expect(wrapper).toMatchSnapshot();
    });

})
