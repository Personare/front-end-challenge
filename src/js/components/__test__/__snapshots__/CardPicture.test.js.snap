// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardPicture should render correctly 1`] = `
<div
  className="CardPicture"
>
  <img
    className="CardPicture__image"
    src="http://via.placeholder.com/162Ã—341"
  />
</div>
`;
